pkg/sdkserver/alpha/localsdk.go

```diff
@@ -14,7 +14,10 @@
 
 package alpha
 
-import "agones.dev/agones/pkg/sdk/alpha"
+import (
+	"agones.dev/agones/pkg/sdk/alpha"
+	"golang.org/x/net/context"
```

Comment:
Is there a reason you can't just import "context" (https://golang.org/pkg/context/)?

Reply:
You know what - my IDE always brought in x/net/context and I never questioned it - I assumed there was a dependency there in gRPC... and I was totally wrong.

Excellent spotting. Fixed!